templates/terraform/examples/app_engine_service_split_traffic.tf.erb

```diff
@@ -0,0 +1,69 @@
+resource "google_storage_bucket" "bucket" {
+	name = "<%= ctx[:vars]['bucket_name'] %>"
+}
+
+resource "google_storage_bucket_object" "object" {
+	name   = "hello-world.zip"
+	bucket = "${google_storage_bucket.bucket.name}"
+	source = "./test-fixtures/appengine/hello-world.zip"
+}
+
+resource "google_app_engine_standard_app_version" "myapp_v1" {
+  version_id = "v1"
+  service = "myapp"
+  runtime = "nodejs10"
+  noop_on_destroy = true
+  entrypoint {
+    shell = "node ./app.js"
+  }
+  deployment {
+    zip {
+      source_url = "https://storage.googleapis.com/${google_storage_bucket.bucket.name}/hello-world.zip"
+    }  
+  }
+  env_variables = {
+    port = "8080"
+  } 
+  depends_on = ["google_storage_bucket_object.object"]
+
+}
+resource "google_app_engine_standard_app_version" "myapp_v2" {
+  version_id = "v2"
+  service = "myapp"
+  runtime = "nodejs10"
+  entrypoint {
+    shell = "node ./app.js"
+  }
+  deployment {
+    zip {
+      source_url = "https://storage.googleapis.com/${google_storage_bucket.bucket.name}/hello-world.zip"
+    }  
+  }
+  env_variables = {
+    port = "8080"
+  } 
+  depends_on = ["google_app_engine_standard_app_version.myapp_v1"]
+}
+
+resource "google_app_engine_service_split_traffic" "myapp" {
+  service_id = "${google_app_engine_standard_app_version.myapp_v2.service}"
+  migrate_traffic = false
+  split {
+    shard_by = "IP"
+    allocations = {
+      v1 = 0.75
```

Comment:
Are you able to interpolate the version names off of the `google_app_engine_standard_app_version` resource?

Reply:
This is unaddressed.

Reply:
Resolving as old.